SConstruct

```diff
@@ -53,15 +53,18 @@ if env["target"] in ["editor", "template_debug"]:
     except AttributeError:
         print("Not including class reference as we're targeting a pre-4.3 baseline.")
 
-file = "{}{}{}".format(libname, env["suffix"], env["SHLIBSUFFIX"])
+# .dev doesn't inhibit compatibility, so we don't need to key it.
+# .universal just means "compatible with all relevant arches" so we don't need to key it.
+suffix = env['suffix'].replace(".dev", "").replace(".universal", "")
+
+lib_filename = "%s%s%s%s".format(env.subst('$SHLIBPREFIX'), libname, suffix, env.subst('$SHLIBSUFFIX'))
 
-libraryfile = "bin/{}/{}".format(env["platform"], file)
 library = env.SharedLibrary(
-    libraryfile,
+    f"bin/{env['platform']}/{lib_filename}",
```

Comment:
Is that possible here as well with `.format`? Just for consistency reasons

Reply:
Sure! Done